app/src/main/java/org/jellyfin/androidtv/ui/GridFragment.java

```diff
@@ -152,8 +152,9 @@ public SortOption(String name, String value, SortOrder order) {
         sortOptions.put(1, new SortOption(TvApp.getApplication().getString(R.string.lbl_date_added), "DateCreated,SortName", SortOrder.Descending));
         sortOptions.put(2, new SortOption(TvApp.getApplication().getString(R.string.lbl_premier_date), "PremiereDate,SortName", SortOrder.Descending));
         sortOptions.put(3,new SortOption(TvApp.getApplication().getString(R.string.lbl_rating), "OfficialRating,SortName", SortOrder.Ascending));
-        sortOptions.put(4,new SortOption(TvApp.getApplication().getString(R.string.lbl_critic_rating), "CriticRating,SortName", SortOrder.Descending));
-        sortOptions.put(5,new SortOption(TvApp.getApplication().getString(R.string.lbl_last_played), "DatePlayed,SortName", SortOrder.Descending));
+        sortOptions.put(4,new SortOption(TvApp.getApplication().getString(R.string.lbl_stars), "CommunityRating,SortName", SortOrder.Descending));
```

Comment:
I'm not sure about the used label, We might want to use "Community Rating" or something like that as label.

Reply:
I can add a new string called lbl_community_rating. To make it consistent i can also change the enum RatingType to those labels. There RATING_TOMATOES with lbl_tomatoes and RATING_STARS with lbl_stars is still used. 
If that would be ok i'm gonna do that.

The jellyfin webui is currently using IMDb Rating and Critic Rating as labels. That would also be an option.

Reply:
I would recommend using the same labels as web for consistency.

Reply:
I think we should not use the imdb name as it's trademarked, same reason that we don't use it when choosing which rating to show on cards.
Besides, the value is not always an imdb rating. different metadata provider plugins can put different values in it.

Reply:
Should we also rename the option in the settings to Critic and Community Rating? Would make it equal everywhere.